Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -338,6 +340,56 @@ static uint32_t convertSystemLayoutDirection(NSUserInterfaceLayoutDirection dire
 
 #endif // PLATFORM(MAC)
 
+- (void)_initializeScreenTimeWebpageController
+{
+#if ENABLE(SCREEN_TIME)
+    if (!_page->preferences().screenTimeEnabled())
+        return;
+    if (!_screenTimeWebpageController) {
+        _screenTimeWebpageController = adoptNS([[STWebpageController alloc] init]);
+        [_screenTimeWebpageController addObserver:self forKeyPath:@"URLIsBlocked" options:(NSKeyValueObservingOptionOld | NSKeyValueObservingOptionNew) context:&readyForDisplayKVOContext];
+#if PLATFORM(MAC)
+        NSView *screenTimeView = [_screenTimeWebpageController view];
+        screenTimeView.frame = self.bounds;
+        [self addSubview:screenTimeView];
+#endif // PLATFORM(MAC)
+        _page->setScreenTimeWebpageController(_screenTimeWebpageController);
+    }
+#endif // ENABLE(SCREEN_TIME)
+}
+
+#if ENABLE(SCREEN_TIME)
+- (void)_deallocScreenTimeWebpageController
```

Comment:
"Dealloc" means something specific, which this doesn't actually do. Maybe "uninstall"?

Reply:
And if you do that maybe the other one becomes "install"?

Reply:
Will change!